src/material-examples/button-types/button-types-example.html

```diff
@@ -72,3 +62,28 @@ <h3>Mini Fab Buttons</h3>
   </button>
   <a mat-mini-fab routerLink=".">Link</a>
 </div>
+
+<h3>Stroked Buttons</h3>
```

Comment:
Can you move the stroked and flat buttons should be above the icon and fab buttons.